backend/portal/admin.py

```diff
@@ -1,19 +1,85 @@
 from django.contrib import admin
 from django.utils.html import escape, mark_safe
 
-from portal.models import Poll, PollOption, PollVote, Post, TargetPopulation
+from portal.models import Content, Poll, PollOption, PollVote, Post, TargetPopulation
 
 
-class PostAdmin(admin.ModelAdmin):
-    def image_tag(self, instance):
-        return mark_safe('<img src="%s" height="300" />' % escape(instance.image.url))
+class ContentAdmin(admin.ModelAdmin):
+    @admin.action(description="Set status to Approved")
+    def action_approved(modeladmin, request, queryset):
+        queryset.update(status=Content.STATUS_APPROVED)
 
-    image_tag.short_description = "Post Image"
-    readonly_fields = ("image_tag",)
+    @admin.action(description="Set status to Draft")
+    def action_draft(modeladmin, request, queryset):
+        queryset.update(status=Content.STATUS_DRAFT)
+
+    @admin.action(description="Set status to Revision")
+    def action_revision(modeladmin, request, queryset):
+        queryset.update(status=Content.STATUS_REVISION)
+
+    actions = [action_approved, action_draft, action_revision]
+
+    def get_queryset(self, request):
+        queryset = super().get_queryset(request)
+        return queryset.annotate(ar=Content.ACTION_REQUIRED_CONDITION).order_by(
+            "-ar", "created_date"
+        )
+
+    def ar(self, obj):
```

Comment:
Seeing as `ar` is displayed in the UI, can we change the UI to show `Action Required` instead for readability at first glance?

Reply:
<img width="1440" alt="Screenshot 2024-04-16 at 10 25 19 AM" src="https://github.com/pennlabs/penn-mobile/assets/72774400/31f476d8-026e-47ca-aa45-1a4250b92bfd">

It takes up so much screen real estate though... so i went against it. What do you think?

Reply:
<img width="1440" alt="Screenshot 2024-04-16 at 10 26 31 AM" src="https://github.com/pennlabs/penn-mobile/assets/72774400/20306fe7-26b3-4cd3-85b7-0a48eb8bc012">

to compare

Reply:
I'm not too against it, but I'm good with whatever you and yoonjung are good with!